helm/charts/dsg-runner-infra/values.yaml

```diff
@@ -0,0 +1,39 @@
+envName:
```

Comment:
I dont think the value 'envName' is used anywhere in the templates.

Reply:
If so we probably want to remove it for all of the other values.yaml files aswell.

Reply:
Good point but this will be replaced by the new ci structure in new infra